pixeltable/tests/test_client.py

```diff
@@ -3,6 +3,84 @@
 import pixeltable as pt
 from pixeltable import exceptions as exc
 
+import pyarrow.parquet as pq
+import pyarrow as pa
+from pixeltable.utils.arrow import iter_tuples
+
+import datasets
+from collections import namedtuple
+
+from datetime import datetime
+from pathlib import Path
+
+def make_test_arrow_table(output_path : Path):
```

Comment:
we already have a test/utils.py with stuff like that

also: missing return type